KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyView.swift

```diff
@@ -162,6 +174,8 @@ final class KarhooLoyaltyView: UIStackView {
     
     override func updateConstraints() {
         if !didSetupConstraints {
+            containerStackView.anchor(top: topAnchor, leading: leadingAnchor, bottom: bottomAnchor, trailing: trailingAnchor, paddingTop: UIConstants.Dimension.View.largeTagHeight / 2)
```

Comment:
I do not know what is the cap of Lint for allowed characters in line, but I think in this case there should be breaking in use.